app/views/complaints/_complaints_table.html.erb

```diff
@@ -16,7 +16,7 @@
   </thead>
   <tbody>
   <% @complaints.each do |complaint| %>
-    <%= render "complaints/complaints_table_detail", complaint: complaint %>
+    <%= render "complaints/complaints_table_detail", complaint_data: complaint %>
```

Comment:
[question] I decided to pass in the data and stick it in `Complaint.new` in the partial, so that it was more clear in the partial what sort of data it was dealing with, but I think either way is probably fine. Any preferences?

Reply:
[thought] i wonder if it makes more sense to take a step up the chain and make `@complaints` a collection of complaint objects. i think that would be my preference. that and making this a collection partial, if possible (i know there are some funky naming conventions that can make that tricky)

Reply:
I thought about that and then decided not to pull on the thread. I mean, presumably that could be happening at the api.request sort of level but I didn't want to have to change things around alllll over the app when I was ostensibly just here to add a Yes for RAN / TTA, lol.  I do think it's a good idea, though.  What do you think?  This PR or should I make an issue for a future one?

Reply:
I think it probably makes the most sense to have the Api::Hses class wrap the response in the complaint objects. I'd vote for doing that here, but I don't mind if it's done in a future one either.

Reply:
Okay. I'll do that here, but leave the other `Api::x` classes alone for now!

Reply:
HMM actually upon further review, I'm having a little trouble figuring out how would be best to go about doing this. Can we merge as is and then open a ticket for thinking about where and how to do it within an Api::Hses type location, or in ApiResponse, etc?  I kinda think it's going to be better to do everything at once rather than just complaints.

Reply:
oh i was just thinking we'd map them in the controller, since we're using the object in the view.